app/src/main/java/com/hiddenramblings/tagmo/MainActivity.java

```diff
@@ -587,11 +588,11 @@ public void onDismissed(Snackbar snackbar, int event) {
         int ssbVisibility = View.GONE;
         int tpVisibility = View.GONE;
         try {
-            long amiiboId = TagUtil.amiiboIdFromTag(currentTagData);
-            if (amiiboId == EditorTP.WOLF_LINK_ID) {
+            long amiiboSeriesIndentifier = (TagUtil.amiiboIdFromTag(currentTagData) & AmiiboSeries.MASK);
+            if (amiiboSeriesIndentifier == EditorTP.TP_IDENTIFIER) {

```

Comment:
This is wrong, Since Wolf Link will probably be the only Amiibo that will work with the Cave of Shadows, even if they add more Twilight Princess Amiibo, we should restrict this to just Wolf Link.

Since you are away, would you like me to change this?

Reply:
Yes, that would be awesome! 
Would you also try the idea @HiddenRambler mentioned in my older PR? 

> A better way would be to check the app id: https://www.3dbrew.org/wiki/Amiibo#Games_using_Amiibo_AppData
> You can see this at 0xb6 with value 0x10110e00

I guess I should just stick to moderating the issue thread. 👍  Haha xD

Reply:
You don't need to give up. There is no urgency for this. you can do this after you finish your holiday. Only way to learn is by doing.

Reply:
Yep, it wasn't wrong as in "this won't do what you intend it to do", it was more "this has a specific condition for it to work and you've broadened that condition so that it may no longer be correct in the future"